environment.yml

```diff
@@ -0,0 +1,17 @@
+name: occ
+channels:
+  - conda-forge # pythonocc-core
+  - defaults
+dependencies:
+  - python=3.8
+  - anaconda
+  - pip
+  - COMPAS
+#  - compas_view2 # could not find a channel that hosts this
```

Comment:
will be on conda-forge but the feedstocks are backed up because of some server error somewhere. should be available soon

Reply:
https://anaconda.org/conda-forge/compas_view2/files
jeej